### ISSUE TITLE
[Entensibility Enhancement] Enum 362 "Analysis Period Type".  Extensible Property 

### ISSUE THREAD

**Issue by MCHammerTNP** [microsoft/ALAppExtensions#16801]
Can the Extensible property be added Enum 362 "Analysis Period Type" so that we can add additional values via an enum extension please.

```
enum 362 "Analysis Period Type"
{
    AssignmentCompatibility = true;
    Extensible = true;

    value(0; "Day") { Caption = 'Day'; }
    value(1; "Week") { Caption = 'Week'; }
    value(2; "Month") { Caption = 'Month'; }
    value(3; "Quarter") { Caption = 'Quarter'; }
    value(4; "Year") { Caption = 'Year'; }
    value(5; "Accounting Period") { Caption = 'Accounting Period'; }
}

```

**Comment by JesperSchulz** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. Please do not reply to this, as we do not monitor closed issues. If you have follow-up questions or requests, please create a new issue where you reference this one.

**Comment by MCHammerTNP** [comment]
Was this ever actioned?

**Comment by MCHammerTNP** [comment]
Hi Jesper,

Was this ever actioned as I can’t see the change in my latest release of 20.8 Build 49971

Rgds
 ​​
Michael Clarendon
Senior Consultant
***@***.***
+442083289818
***@***.***
07436 144708

***@***.***<https://www.facebook.com/thenavpeople>
***@***.***<https://www.linkedin.com/company/the-nav-365-people>
***@***.***<https://www.twitter.com/TheNAVPeople>
***@***.***<https://www.youtube.com/channel/UC4D1Fxl_SwyFX7abeIjaI-A>
Disclaimer: www.the365people.com/disclaimer<https://the365people.com/disclaimer>
***@***.***
Cyber Essentials Plus<https://www.cyberessentials.ncsc.gov.uk/about>

**Comment by JesperSchulz** [comment]
It is addressed from 21.0:

![image](https://user-images.githubusercontent.com/19796701/207266535-3b9cad1b-287b-440f-b444-b2485741abc4.png)

**Comment by MCHammerTNP** [comment]
Hi Jesper,

Thanks for confirming it is in version 21.0

Rgds
 ​​
Michael Clarendon
Senior Consultant
***@***.***
+442083289818
***@***.***
07436 144708

***@***.***<https://www.facebook.com/thenavpeople>
***@***.***<https://www.linkedin.com/company/the-nav-365-people>
***@***.***<https://www.twitter.com/TheNAVPeople>
***@***.***<https://www.youtube.com/channel/UC4D1Fxl_SwyFX7abeIjaI-A>
Disclaimer: www.the365people.com/disclaimer<https://the365people.com/disclaimer>
***@***.***
Cyber Essentials Plus<https://www.cyberessentials.ncsc.gov.uk/about>
